packages/stark-core/src/modules/user/services/user.service.ts

```diff
@@ -79,7 +78,7 @@ export class StarkUserServiceImpl implements StarkUserService {
 
 		const allUsers: StarkUser[] = this.userProfiles;
 
-		if (!allUsers || allUsers.length === 0) {
+		if (allUsers.length === 0) {
```

Comment:
We should keep the condition as before :
```js
if (!allUsers || allUsers.length === 0)
```

Because when we deserialize the object in the constructor, we are not sure that the deserialization will be fine.
It could return `null` or `undefined` and so we'll have the error `cannot read length of undefined`...

Reply:
I've updated this to fallback to `[]` in case the deserialization returns `null` or `undefined`... so we always return something ;)